it/controllers/benefits/income/IncomeTaxBenefitsAmountControllerISpec.scala

```diff
@@ -0,0 +1,546 @@
+            dropEmploymentDB()
+            userDataStub(userData(fullEmploymentsModel(hmrcEmployment = Seq(employmentDetailsAndBenefits(fullBenefits)))), nino, taxYearEOY)
+            insertCyaData(employmentUserData(hasPriorBenefits = true, cyaModel(
+              Some(benefits(fullIncomeTaxOrIncurredCostsModel)))), userRequest)
+            urlGet(pageUrl(taxYearEOY), welsh = user.isWelsh, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+          }
+
+          s"has an OK($OK) status" in {
+            result.status shouldBe OK
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+          titleCheck(user.specificExpectedResults.get.expectedTitle)
+          h1Check(user.specificExpectedResults.get.expectedHeading)
+          captionCheck(expectedCaption, captionSelector)
+          textOnPageCheck(optionalParagraphText(amount), paragraphSelector(2))
+          textOnPageCheck(enterTotalText, paragraphSelector(3))
+          textOnPageCheck(expectedHintText, hintTextSelector)
+          inputFieldCheck(amountFieldName, amountFieldSelector)
+          inputFieldValueCheck(amount.toString(), amountFieldSelector)
+          buttonCheck(continueButtonText, continueButtonSelector)
+          formPostLinkCheck(formLink, formSelector)
+
+          welshToggleCheck(user.isWelsh)
+        }
+
+        "render the income tax benefits amount page with a pre-filled amount field when there's cya data and no prior benefits" which {
+          lazy val result: WSResponse = {
+            authoriseAgentOrIndividual(user.isAgent)
+            dropEmploymentDB()
+            insertCyaData(employmentUserData(hasPriorBenefits = false, cyaModel(
+              Some(benefits(fullIncomeTaxOrIncurredCostsModel)))), userRequest)
+            urlGet(pageUrl(taxYearEOY), welsh = user.isWelsh, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+          }
+
+          s"has an OK($OK) status" in {
+            result.status shouldBe OK
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+          titleCheck(user.specificExpectedResults.get.expectedTitle)
+          h1Check(user.specificExpectedResults.get.expectedHeading)
+          captionCheck(expectedCaption, captionSelector)
+          textOnPageCheck(optionalParagraphText(amount), paragraphSelector(2))
+          textOnPageCheck(enterTotalText, paragraphSelector(3))
+          textOnPageCheck(expectedHintText, hintTextSelector)
+          inputFieldCheck(amountFieldName, amountFieldSelector)
+          inputFieldValueCheck(amount.toString(), amountFieldSelector)
+          buttonCheck(continueButtonText, continueButtonSelector)
+          formPostLinkCheck(formLink, formSelector)
+
+          welshToggleCheck(user.isWelsh)
+        }
+      }
+    }
+
+    val user = UserScenario(isWelsh = false, isAgent = false, CommonExpectedEN, Some(ExpectedIndividualEN))
```

Comment:
this variable isnt used if you want to remove it

Reply:
removed variable